bin/twistcli_validator.py

```diff
@@ -0,0 +1,26 @@
+import json
+import sys
+import os
+from pathlib import Path
+
+
+ROOT_DIRECTORY = Path(__file__).parent.parent
+PROJECT_NAME = os.getenv("PROJECT_DIRECTORY")
+SCAN_RESULTS_FILE_PATH = f"{ROOT_DIRECTORY}/{PROJECT_NAME}"
+output = []
+
+with open(f"{SCAN_RESULTS_FILE_PATH}/scan-results.json", "r") as f:
+    data = json.load(f)
+    vulnerabilities = data.get("results", [])[0].get("vulnerabilities", [])
+    output = [vul["id"] for vul in vulnerabilities]
+
+try:
+    with open(f"{SCAN_RESULTS_FILE_PATH}/twistcliignore", "r") as f:
+        ignore_list = [line.strip() for line in f.read().splitlines()]
+        if ignore_list == output:
+            print("vuln is in ignore list")
+        else:
+            print("New vulnerabilities found.Fix them or add into ignore list")
```

Comment:
We should print the CVEs that are not ignored. We can do this with `set`:
```
>>> ignore_list = ['CVE-2023-39320', 'CVE-2023-29405', 'CVE-2023-29404', 'CVE-2023-29402']
>>> cves = ['CVE-2023-39320', 'CVE-2023-29405', 'CVE-2023-29404', 'CVE-2023-29402', 'CVE-2023-29403', 'CVE-2023-39533', 'CVE-2023-39322', 'CVE-2023-39321']
>>> set(cves) - set(ignore_list)
{'CVE-2023-39321', 'CVE-2023-39533', 'CVE-2023-39322', 'CVE-2023-29403'}
```

Or with a comprehension:

```
>>> [cve for cve in cves if cve not in ignore_list]
['CVE-2023-29403', 'CVE-2023-39533', 'CVE-2023-39322', 'CVE-2023-39321']
```